recipes/rptools/meta.yaml

```diff
@@ -0,0 +1,70 @@
+{% set name = "rptools" %}
+{% set version = "5.8.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  path: ..
```

Comment:
Why do you need this? It is more transparent to write a multi-line build script that CD into the directory,

Reply:
Actually, I don't need it, I just forgot to delete it. I just did ! Thank you @ocefpaf !